integreat_cms/cms/constants/language_color.py

```diff
@@ -0,0 +1,90 @@
+"""
+This module contains constants for Language model.
+"""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from django.utils.translation import gettext_lazy as _
+
+from ..utils.translation_utils import gettext_many_lazy as __
+
+if TYPE_CHECKING:
+    from typing import Final
+
+    from django.utils.functional import Promise
+
+MELLOW_APRICOT: Final = "#FFBB78"
+FOREST_GREEN: Final = "#2CA02C"
+ROSE: Final = "#FF9896"
+TROPICAL_VIOLET: Final = "#C5B0D5"
+RED: Final = "#FF4500"
+ORANGE: Final = "#FFA500"
+DARK_BLUE: Final = "#17157D"
+GREEN_BLUE: Final = "#1F77B4"
+YELLOW: Final = "#FFD700"
+TEAL: Final = "#008080"
+ARCTIC: Final = "#9EDAE5"
+AZURE: Final = "#5894E3"
+PACIFIC_BLUE: Final = "#17BECF"
+ORANGE_RED: Final = "#FF6347"
+LIGHT_GREEN: Final = "#98DF8A"
+VIOLET: Final = "#9467BD"
+LIME: Final = "#ADFF2F"
+LAVENDER: Final = "#E377C2"
+BROWN: Final = "#8C564B"
+PINK_ORANGE: Final = "#FFA07A"
+PASTEL_PINK: Final = "#FFE4F0"
+KHAKI: Final = "#F0E68C"
+YELLOW_GREEN: Final = "#BCBD22"
+MAUVE: Final = "#800080"
+PURPLE: Final = "#BA55D3"
+PRIMROSE: Final = "#DBDB8D"
+FIORD: Final = "#4B5563"
+QUICKSAND: Final = "#C49C94"
+GREY: Final = "#7F7F7F"
+AQUA: Final = "#26FCFF"
+PINE_GREEN: Final = "#20B2AA"
```

Comment:
```suggestion
LIGHTSEAGREEN: Final = "#20B2AA"
```
Pine green usually is darker (around #26392f). When I googled this color I got lightseagreen as a name and "Helles Seegrün" for the German translation

Reply:
I have doule-checked and you are 100% right. I was sure that I used the correct name with the help of color code identifier. :)